test-cases/tools/lib/test-case.ts

```diff
@@ -71,8 +79,8 @@ function extractTitle(content: string): { title: string; content: string } {
 }
 
 function extractId(title: string): { id: string; title: string } {
-    // A01 - Title
-    const match = /^(?<id>[A-Z][0-9]{2})\s-\s(?<title>.*)$/.exec(title);
+    // A01a - Title
+    const match = /^(?<id>[A-Z][0-9]{2}[a-z]?)\s-\s(?<title>.*)$/.exec(title);
```

Comment:
Because we have only `a` or `b` as post id letter we could updated the regex to include only them, also it would be nice to have the id all uppercase , like this:
```suggestion
    const match = /^(?<id>[A-Z][0-9]{2}[AB]?)\s-\s(?<title>.*)$/.exec(title);
```